File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/attivita/AttivitaCDG.ts
import { Job, PKJob } from './Job';
import { TipiJob } from './TipiJob';

export class AttivitaCDG extends Job {
    constructor(codice: string) {
        super({classType:'AttivitaCDG',codice:codice} as PKJob);
    }

    getTipo(): TipiJob {
        return TipiJob.ATTIVITACDG;
    }


}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/attivita/AttivitaCDG.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.attivita
{
    public class AttivitaCDG : Job
    {
        public AttivitaCDG()
        {
            tipo = TipiJob.ATTIVITACDG;
            primaryKey = new PK(costanti.ClassType.AttivitaCDG, "0");
        }

        public AttivitaCDG(int codice, string descrizione)
        {
            tipo = TipiJob.ATTIVITACDG;
            primaryKey = new PK(costanti.ClassType.AttivitaCDG, codice.ToString());
            this.descrizione = descrizione;
        }
    }
}
